README.md

```diff
@@ -53,7 +53,7 @@ See the [repoquery documentation](https://mamba.readthedocs.io/en/latest/user_gu
 
 ### provision-with-micromamba (setup-miniconda replacement)
 
-[provision-with-micromamba](https://github.com/marketplace/actions/provision-with-micromamba) is a replacement for [setup-miniconda](https://github.com/marketplace/actions/setup-miniconda) that uses `micromamba`.
+[setup-micromamba](https://github.com/marketplace/actions/setup-micromamba) is a replacement for [setup-miniconda](https://github.com/marketplace/actions/setup-miniconda) that uses `micromamba`.
```

Comment:
I wonder if we could work with GH's dependabot to help with this migration. Tons of repos will be stuck in the "provision-with-micromamba" one.

Reply:
That'd be great, also for updating projects `environment.yml` files.